server/controllers/ImageControllers.js

```diff
@@ -0,0 +1,72 @@
+const AWS = require("aws-sdk");
+const multer = require("multer");
+const { v4: uuidv4 } = require("uuid");
+const NodeCache = require("node-cache");
+
+const imgCache = new NodeCache();
+
+const s3 = new AWS.S3({
+  accessKeyId: process.env.AWS_ID,
+  secretAccessKey: process.env.AWS_SECRET,
+});
+
+const storage = multer.memoryStorage({
+  destination: function (req, file, cb) {
+    cb(null, "");
+  },
+});
+
+const upload = multer({ storage }).single("profilePicture");
+
+const uploadImage = async (req, res) => {
+  const fileNameParts = req.file.originalname.split(".");
+  const fileType = fileNameParts[fileNameParts.length - 1];
+
+  const key = `${uuidv4()}.${fileType}`;
+
+  const params = {
+    Bucket: process.env.AWS_BUCKET_NAME,
```

Comment:
Can you send the AWS environment variables in the Slack channel, I'll need them to test

Reply:
Sure, I will send them today.